### DISCUSSION TITLE
Simplest example not working with 0.42 on osx

### DISCUSSION THREAD

**Discussion by porttack** [k14s/ytt#715] [General]
x.yaml:
```
a: 1
b: 2
```

Now run `ytt --data-value a=3 -f x.yaml` and I still see `a: 1`. Seems like a great tool, but I'm at a loss.

**Comment by mamachanko** [comment]
[Data values](https://carvel.dev/ytt/docs/v0.42.0/how-to-use-data-values/) are what you want. Try this:
```yaml
#! x.yaml

#@ load("@ytt:data", "data")

a: #@ data.values.a
```

Now, run `ytt --data-value a=3 -f x.yaml` and you should see `a: 3`.
